hoot-core/src/main/cpp/hoot/core/algorithms/AlphaShape.cpp

```diff
@@ -327,8 +327,11 @@ void AlphaShape::insert(const vector< pair<double, double> >& points)
   {
     if (i % 10000 == 0 && Log::getInstance().getLevel() <= Log::Info)
     {
+      //wanted to convert this logging to trace, but saw some strange screen output behavior when
+      //doing so...will look at later
```

Comment:
I think using the LOG_TRACE macro could get you in trouble here, because it likely drops a "\n" or something at the end of the message, which will mess up the use of the "\r" in this output. Best solution might be to wrap this in an if-block, something like if (TRACE == hoot::Log::getInstance().getLevel())